README.md

```diff
@@ -131,6 +131,13 @@ We pass `--acount-contract` to tell the verifier that the contract is of type ac
 
 For `<LICENSE_SCHEME>` the command takes [_No License (None)_](https://github.com/github/choosealicense.com/blob/a40ef42140d137770161addf4fefc715709d8ccd/no-permission.md) as default license scheme. [Here](https://goerli.voyager.online/cairo-licenses) is a list of available options.
 
+### `starknet-new-account`
+
+```
+npx hardhat starknet-new-account [--starknet-network <NAME>] [--wallet <WALLET_NAME>]
+```
+
+Initializes a wallet `wallets["WALLET_NAME"]` configured in the `hardhat.config` file. _Needs to be funded before deploying it._ Uses the modified OZ implementation used by StarkNet CLI.
```

Comment:
This part of docs can mention needing to use `starknet-deploy-account`, and then `starknet-deploy-account` docs can mention that it needs to be funded before deploying.

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/246/commits/368bcfb2129f10a4091654e14a60550131d5117e